configs/envoy_double_proxy_v2.template.yaml

```diff
@@ -0,0 +1,102 @@
+{%- macro listener(address,port_value,proxy_proto) -%}
+- name: listener_created_from_configgen
+  address:
+    socket_address:
+      address:  {{address}}
+      port_value: {{port_value}}
+  filter_chains:
+    - filter_chain_match:
+        sni_domains:  example.com
+      filters:
+        name: envoy.http_connection_manager
+        config:
+          codec_type: AUTO
+          stat_prefix:  router
+          route_config:
+            name: route_configuration_for_http_connection_manager
+            virtual_hosts:
+              - name: all
+                domains:
+                  - *
+                routes:
+                  - match:
+                      prefix: "/"
+                    route:
+                      cluster:  backhaul
+                      timeout:  20s
+          http_filters:
+            - name: envoy.health_check
+              config:
+                pass_through_mode: false
+                endpoint: /healthcheck
+            - name: envoy.buffer
+              config: 
+                max_request_bytes: 5242880
+                max_request_time: 120
+            - name: envoy.router
+              config: {}
+          access_log:
+            - name: envoy.file_access_log
+              config:
+          {% if proxy_proto %}
+          use_remote_address: true
+          {%endif -%}
+     {% if proxy_proto %}
+      use_proxy_proto: true
+     {%endif -%}
+{% endmacro -%}
+admin:
+  access_log_path:  "var/log/envoy/admin_access.log"
+  address:
+    socket_address:
+      protocol:
+      address:  tcp://127.0.0.1
+      port_value: 9901
+      named_port:
+      resolver_name:
+      ipv4_compat:    
+flags_path: /etc/envoy/flags
+stats_sinks:
+  name: envoy.statsd
+  config: 
+    tcp_cluster_name: statsd
+tracing:
+  http:
+    name: envoy.lightstep
+    config:
+      collector_cluster:  etc/envoy/lightstep_access_token
+      access_token_file:  lightstep_saas
+runtime:
+  symlink_root: /srv/runtime_data/current
+  subdirectory: envoy
+  override_subdirectory:  envoy_override
```

Comment:
nit: not sure why, but there are many fields like this that have two spaces, can you fix?